Crypto-Currency/Crypto-Currency/Scenes/Ranking/RankingViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  RankingViewController.swift
+//  Crypto-Currency
+//
+//  Created by namtrinh on 04/08/2021.
+//
+
+import UIKit
+
+final class RankingViewController: UIViewController {
+
+    @IBOutlet private weak var rankingTableView: UITableView!
+    
+    private var coins = [Coin]()
+    private let repository = RepositoryAPI()
+    private var offset = 30
+    private var isLoading = false
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        loadAPI()
+    }
+    
+    private func configureTableView() {
+        rankingTableView.delegate = self
+        rankingTableView.dataSource = self
+        rankingTableView.register(CoinTableViewCell.nib, forCellReuseIdentifier: CoinTableViewCell.reuseIdentifier)
+    }
+    
+    private func loadAPI() {
+        repository.getCoins(urlString: EndPoint.ranking.rawValue) { [weak self] coins, error in
+            guard let self = self else { return }
+            if let error = error {
+                print(error)
```

Comment:
không xử lý lỗi mà chỉ in ra thôi hả em?

Reply:
Hay là show alert ạ @@. E chưa nghĩ ra nên làm gì ở trong trường hợp này ạ.

Reply:
À em vừa xử lý r a check giúp em với xem đúng k ạ :D